hkdf.go

```diff
@@ -13,7 +13,12 @@ import (
 )
 
 func SupportsHKDF() bool {
-	return versionAtOrAbove(1, 1, 1)
+	ctx := C.go_openssl_EVP_PKEY_CTX_new_id(C.GO_EVP_PKEY_HKDF, nil)
+	if ctx == nil {
+		return false
+	}
+	C.go_openssl_EVP_PKEY_CTX_free(ctx)
+	return false
```

Comment:
I must be missing something... shouldn't this func this return `true` sometimes?

Reply:
Wops, good catch.